### DISCUSSION TITLE
In need of help

### DISCUSSION THREAD

**Discussion by WildCardM** [geocine/sd-easy-mode#27] [Q&A]
Wanted to say that I made an account just to post on this, its the best dreambooth i found and it makes me quite sad when I can't get it to work after updates. Usually I can get it back up and running by searching the errors I find around but this time...I'm in a pickle so I thought to try and ask the creator himself for help to make my life easier.

So the first error I get: attributeerror: module 'jax.random' has no attribute 'keyarray' Which I fixed using 
!pip install "jax[cuda12_pip]==0.4.23" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

And that worked for a while, but nowadays I'm getting a second error: Exception: CUDA SETUP: Setup Failed!

**Comment by geocine** [comment]
Thanks for the compliment! I have both good and bad news for you. The bad news is that I’ve paused work on this project and Stable Diffusion 1.5 for quite a while now. I might return to it at some point, but even the original author of the base code it relies on has also stopped working on the project. The good news is I have a new tool you can explore that's in a similar vein but offers even better results than we typically achieved with Stable Diffusion 1.5. I’ve created a repository for Flux, and if you haven’t come across it yet, I recommend checking it out. Here’s the link to my tutorial and video: https://github.com/geocine/flux.
